diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/kdoc/KdocMethods.kt

```diff
@@ -55,9 +57,14 @@ import org.jetbrains.kotlin.kdoc.psi.impl.KDocTag
  */
 @Suppress("ForbiddenComment")
 class KdocMethods : Rule("kdoc-methods") {
-    // expression body of function can have a lot of 'ElementType's, this list might be not full
-    private val expressionBodyTypes = setOf(BINARY_EXPRESSION, CALL_EXPRESSION, LAMBDA_EXPRESSION, REFERENCE_EXPRESSION,
-            CALLABLE_REFERENCE_EXPRESSION, SAFE_ACCESS_EXPRESSION, WHEN_CONDITION_WITH_EXPRESSION, COLLECTION_LITERAL_EXPRESSION)
+    companion object {
+        // expression body of function can have a lot of 'ElementType's, this list might be not full
+        private val expressionBodyTypes = setOf(BINARY_EXPRESSION, CALL_EXPRESSION, LAMBDA_EXPRESSION, REFERENCE_EXPRESSION,
+                CALLABLE_REFERENCE_EXPRESSION, SAFE_ACCESS_EXPRESSION, WHEN_CONDITION_WITH_EXPRESSION, COLLECTION_LITERAL_EXPRESSION)
+
+        // these methods do not need mandatory documentation
+        private val standardMethods = listOf("equals", "hashCode", "toString", "clone", "finalize")
```

Comment:
how about c o n f i g u r a t i o n?

Reply:
at least put FixMe

Reply:
You mean, someone could possibly want to make documentation of `equals` mandatory, or that user should be able to add their own methods which do not need to be documented? In the latter case, we could have two separate configuration options: `standard methods` and `custom methods`.

Reply:
Eventually decided not to allow customizations here.